ReactAndroid/src/main/java/com/facebook/react/ReactInstanceManager.java

```diff
@@ -230,24 +229,19 @@ private static void initializeSoLoaderIfNecessary(Context applicationContext) {
    * {@link ReactRootView} is available and measured.
    */
   public void createReactContextInBackground() {
-    if (mUseDeveloperSupport) {
+    if (mUseDeveloperSupport && mJSMainModuleName != null) {
```

Comment:
We do want to change to falling back to the bundle on disk if the server isn't available, but that can happen in another patch.